IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -200,9 +200,11 @@ block ReaderTMY3 "Reader for TMY3 weather data"
         iconTransformation(extent={{-240,-240},{-200,-200}})));
 
   //--------------------------------------------------------------
-  parameter String filNam="" "Name of weather data file" annotation (Dialog(
+  parameter String filNam="modelica://IBPSA/Resources/weatherdata/USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos" "Name of weather data file" annotation (Dialog(
```

Comment:
Is this intentional and required?

Reply:
At this moment it is required, since I thought this will solve the errors regarding the "invalid connection blocks". However the problem persists when exporting FMUs, this is why some tests still fail. I'll have to think of something else, but I'm open to ideas.